apps/pre-award/copilot/environments/addons/sanitised-database-dumps.yml

```diff
@@ -0,0 +1,75 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+    Default: dev
```

Comment:
don't think we should have a default value here? it always gets injected by copilot